lightning/src/chain/keysinterface.rs

```diff
@@ -208,6 +236,9 @@ pub trait ChannelKeys : Send+Clone {
 	fn commitment_seed<'a>(&'a self) -> &'a [u8; 32];
 	/// Gets the local channel public keys and basepoints
 	fn pubkeys<'a>(&'a self) -> &'a ChannelPublicKeys;
+	/// Gets the channel key state for rederivation, namely chan_id, starting_time_nanos,
```

Comment:
Please don't expose this with the exact names of how we use them in KeysManager - its up to the user how they want to use them, they're just user_id_1, and user_id_2.